packages/core/src/components/content-header/content-header.scss

```
@@ -26,6 +28,17 @@
       text-overflow: ellipsis;
     }
 
+    .header-title {
+      @include fonts.typography-h3;
+      margin: 0;
+      padding: 0;
+
+      &.secondary {
+        @include fonts.typography-h4;
+        padding: 0.25rem 0;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)